packages/http-client-python/emitter/src/http.ts

```diff
@@ -338,7 +337,7 @@ function emitHttpBodyParameter(
 function emitHttpResponse(
   context: PythonSdkContext<SdkHttpOperation>,
   statusCodes: HttpStatusCodeRange | number | "*",
-  response: SdkHttpResponse | SdkHttpErrorResponse,
+  response: SdkHttpResponse,
```

Comment:
this is for adopting latest tcgc version, is it a false change?

Reply:
apparently `SdkHttpErrorResponse` got removed from tcgc, not sure what happened there so I just removed it to keep it up to date. But let me take a closer look

Reply:
this is added in 0.46.2, refer this pr: https://github.com/Azure/typespec-azure/pull/1589/files#diff-12c770b9ca64a6c3884d87ed02dbc3afa28ccfc2a07d258459342260948004e9